domains/authorization/Authorization.API/API/Program.cs

```diff
@@ -14,9 +14,9 @@
 var otlpConfiguration = builder.Configuration.GetSection(OtlpOptions.Prefix);
 var otlpOptions = otlpConfiguration.Get<OtlpOptions>()!;
 
-builder.AddSerilogWithOpenTelemetry(otlpOptions.ReceiverEndpoint);
+builder.AddSerilog();
```

Comment:
Don't use setup anymore?

Reply:
Yes, But the Method in the setup package was changed, in version 4.0.0, to no longer export logs to opentelemetry. They will now be scraped by PromTail